### PR TITLE
WW-367: Style checkboxes in Create New Wiki

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
Positioning label for checkbox "Is this wiki intended for kids?" should be done in ticket with tooltip.

@Wikia/west-wing

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/84906d0611d3e8c70cf17d1ac3f28ed96557591c/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/84906d0611d3e8c70cf17d1ac3f28ed96557591c/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L457](https://github.com/Wikia/app/blob/84906d0611d3e8c70cf17d1ac3f28ed96557591c/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L457): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. MAJOR [WikiBuilder.js#L591](https://github.com/Wikia/app/blob/84906d0611d3e8c70cf17d1ac3f28ed96557591c/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L591): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L593](https://github.com/Wikia/app/blob/84906d0611d3e8c70cf17d1ac3f28ed96557591c/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L593): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 1]
```diff
@@ -162,19 +162,57 @@
...
+					left: 4px;
+					opacity: 0;
+					position: absolute;
+					top: 4px;
+					transform: rotate(-45deg) translate3d(0,0,0);
+					width: 8px;
+				}
+
+				&:hover::after {
+					opacity: 0.3;
```
Comment (Warkot) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#leadingzero

**Review by rybmat — COMMENTED** [review]
there are some magic numbers here, did you considered moving them to variables?

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 2]
```diff
@@ -162,19 +162,57 @@
...
+				border: 1px solid $wds-color-light-gray;
+				border-radius: 1px;
+				cursor: pointer;
+				height: 18px;
+				left: 0;
+				position: absolute;
+				top: 0;
+				width: 18px;
+
+				&:after {
```
Comment (Warkot) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: Double colon (`&::after {`) is preferred for pseudo-elements. https://developer.mozilla.org/en/docs/Web/CSS/Pseudo-elements#Notes

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 3]
```diff
@@ -162,19 +162,57 @@
...
+					background-color: transparent;
+					border: 2px solid $wds-color-white;
+					border-top: none;
+					border-right: none;
```
Comment (Warkot) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: `border: 0;` > `border: none;` https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#borderzero

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 4]
```diff
@@ -162,19 +162,57 @@
...
+					content: '';
+					height: 4px;
+					left: 4px;
+					opacity: 0;
+					position: absolute;
+					top: 4px;
+					transform: rotate(-45deg) translate3d(0,0,0);
```
Comment (Warkot) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: I saw recently `transform` property being browser-prefixed. However, http://caniuse.com/#search=transform shows that prefixes are not needed for browsers we support. We're dropping the mixin? I'd say yes, but wanted to be sure. :)

**Comment by Warkot** [comment]
Some minor comments, besides them LGTM.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/f723945e04cbcdb0444ddbbbdbb50a88c5d475fc/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/f723945e04cbcdb0444ddbbbdbb50a88c5d475fc/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L457](https://github.com/Wikia/app/blob/f723945e04cbcdb0444ddbbbdbb50a88c5d475fc/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L457): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. MAJOR [WikiBuilder.js#L591](https://github.com/Wikia/app/blob/f723945e04cbcdb0444ddbbbdbb50a88c5d475fc/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L591): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L593](https://github.com/Wikia/app/blob/f723945e04cbcdb0444ddbbbdbb50a88c5d475fc/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L593): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/05de9b0f8f50de372ed44d43060843acb94578d4/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/05de9b0f8f50de372ed44d43060843acb94578d4/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L472](https://github.com/Wikia/app/blob/05de9b0f8f50de372ed44d43060843acb94578d4/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L472): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. MAJOR [WikiBuilder.js#L606](https://github.com/Wikia/app/blob/05de9b0f8f50de372ed44d43060843acb94578d4/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L606): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L608](https://github.com/Wikia/app/blob/05de9b0f8f50de372ed44d43060843acb94578d4/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L608): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

**Review by rybmat — APPROVED** [review]

**Review by Warkot — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/2b51d8e20c1d5b076af109c4e842162103a0905e/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
2. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/2b51d8e20c1d5b076af109c4e842162103a0905e/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
3. MAJOR [WikiBuilder.js#L472](https://github.com/Wikia/app/blob/2b51d8e20c1d5b076af109c4e842162103a0905e/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L472): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
4. MAJOR [WikiBuilder.js#L606](https://github.com/Wikia/app/blob/2b51d8e20c1d5b076af109c4e842162103a0905e/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L606): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
5. MAJOR [WikiBuilder.js#L608](https://github.com/Wikia/app/blob/2b51d8e20c1d5b076af109c4e842162103a0905e/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L608): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

**Pull request merged**